apps/common-lib/src/main/scala/com/gu/typerighter/lib/JsonHelpers.scala

```diff
@@ -14,6 +14,9 @@ object JsonHelpers {
   def toJsonSeq[T](serializable: T)(implicit tjs: Writes[T]) =
     Json.toJson(serializable).toString() + recordSeparatorChar
 
+  def toNewlineDeliniatedJson[T](serializable: T)(implicit tjs: Writes[T]) =
```

Comment:
Nit: typos in a spellchecker mean double irony points 

```suggestion
  def toNewlineDelineatedJson[T](serializable: T)(implicit tjs: Writes[T]) =
```